standalone.yml

```diff
@@ -25,6 +25,7 @@ genesis-operations:
           feeer:
             type: fixed
             amount: 1
+            exchange-min-amount: 1
```

Comment:
@wyuinche 
This value is necessary because that currency can be exchanged for another currency. In this case, the value that indicates how much that currency should be exchanged is exchange-min-amount.